package.json

```diff
@@ -550,6 +550,7 @@
     "eslint-plugin-prettier": "^3.0.1",
     "eslint-plugin-promise": "^4.0.1",
     "eslint-plugin-standard": "^4.0.0",
+    "fs-extra": "^8.0.1",
```

Comment:
if `fs-extra` doesn't pollute `fs`, this is fine.  in a perfect world, I would prefer individual modules to handle whatever operations we need, but not willing to die on that hill

Reply:
`fs-extra` does not pollute `fs`.